src/Features/VisualBasic/Portable/ChangeSignature/VisualBasicChangeSignatureService.vb

```diff
@@ -524,6 +518,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ChangeSignature
             Return False
         End Function
 
+        Private Sub GetArgumentListDetails(
+            argumentList As ArgumentListSyntax,
+            ByRef argumentCount As Integer,
+            ByRef lastArgumentIsNamed As Boolean,
+            ByRef lastArgumentExpression As ExpressionSyntax)
```

Comment:
maybe doc this.   for example, why do you need the last last argexpr?

Reply:
Updated method name for clarity